src/VisualStudio/Core/Def/Implementation/TableDataSource/MiscellaneousDiagnosticListTable.cs

```diff
@@ -1,37 +1,31 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.ComponentModel.Composition;
 using System.Linq;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
-using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.TableManager;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.TableDataSource
 {
-    using Workspace = Microsoft.CodeAnalysis.Workspace;
-
-    [Export(typeof(MiscellaneousDiagnosticListTable))]
-    internal class MiscellaneousDiagnosticListTable : VisualStudioBaseDiagnosticListTable
+    internal sealed class MiscellaneousDiagnosticListTable : VisualStudioBaseDiagnosticListTable
```

Comment:
Other than the type being taken in the Register method, what makes this special to the MiscellaneousFilesWorkspace? It all looks pretty general?